src/python/espressomd/interactions.pxd

```diff
@@ -312,6 +344,12 @@ cdef extern from "object-in-fluid/oif_global_forces.hpp":
     int oif_global_forces_set_params(int bond_type, double A0_g, double ka_g, double V0, double kv)
 cdef extern from "object-in-fluid/oif_local_forces.hpp":
     int oif_local_forces_set_params(int bond_type, double r0, double ks, double kslin, double phi0, double kb, double A01, double A02, double kal)
+cdef extern from "immersed_boundary/ibm_triel.hpp":
+    int IBM_Triel_SetParams(const int bond_type, const int ind1, const int ind2, const int ind3, const double max, const tElasticLaw elasticLaw, const double k1, const double k2)
+cdef extern from "immersed_boundary/ibm_tribend.hpp":
+    int IBM_Tribend_SetParams(const int bond_type, const int ind1, const int ind2, const int ind3, const int ind4, const double kb, const bool flat)
+cdef extern from "immersed_boundary/ibm_volume_conservation.hpp":
+    int IBM_VolumeConservation_SetParams(const int bond_type, const int softID, const double kappaV);
```

Comment:
These maybe need to be ifdef'ed to avoid linking errors?  What happens when you build without `#define IMMERSED_BOUNDARY`?